modules/tensor_mechanics/include/materials/cohesive_zone_model/BiLinearMixedMode.h

```diff
@@ -0,0 +1,89 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "CZMComputeLocalTractionTotalBase.h"
+
+/**
+ * Implementation of the mixed mode bilinear traction separation law
+ * described in Mixed-Mode Decohesion Finite Elements for the Simulation of Delamination in
+ *Composite Materials, Pedro P. Camanho and Carlos G. Davila, NASA/TM-2002-211737
+ **/
+class BiLinearMixedMode : public CZMComputeLocalTractionTotalBase
```

Comment:
Should we rename`BiLinearMixedMode` to `CZMBiLinearMixedMode` to be consisten with the rest of CZM materials?